include/stout/flags/flags.cc

```diff
@@ -258,19 +290,15 @@ void Parser::Parse(int* argc, const char*** argv) {
           break;
         }
       } else {
-        // It might be a positional argument or an unknown argument.
-        // If unknown - just exit.
-        std::cerr << "Encountered unknown argument '"
-                  << arg << "'" << std::endl;
-        std::exit(1);
+        // It might be a positional argument. Anyway, we store the name
+        // and empty value in order to try to parse this argument as a
+        // positional one [Parse(values) - function call below].
+        name = arg;
+        values.emplace_back(ArgumentInfo{name, value});
```

Comment:
Generally, if you're calling a constructor anyways, prefer `push_back` over `emplace_back`: https://abseil.io/tips/112

Here and below.